code/modules/clothing/suits/marine_armor.dm

```diff
@@ -382,9 +382,10 @@
 /obj/item/clothing/suit/storage/marine/smartgunner/veteran/PMC
 	name = "\improper PMC gunner armor"
 	desc = "A modification of the standard Armat Systems M3 armor. Hooked up with harnesses and straps allowing the user to carry an M56 Smartgun."
```

Comment:
```suggestion
	desc = "A modification of the standard Armat Systems M3 armor. Hooked up with harnesses and straps allowing the user to carry an T90A Smartgun."
```
I guess M3 Armat armor is still in the lore?

Reply:
oh heck

Reply:
i changed the lore and made it generic since i'm going to change the pmc sgs to be the smartmachinegun later in another PR